pool/payment.go

```diff
@@ -162,3 +162,40 @@ func (pmt *Payment) Archive(db *bolt.DB) error {
 		return nil
 	})
 }
+
+// fetchOrphanedPayments returns all payments sourcing from orphaned blocks at
+// the provided height.
+func fetchOrphanedPayments(db *bolt.DB, height uint32) ([]*Payment, error) {
```

Comment:
Would suggest this be renamed to `pendingPaymentsAtHeight` since at this point the payments returned have not yet been confirmed as being sourced from orphaned blocks.

Reply:
Done